numpy/random/mtrand.pyx

```diff
@@ -3703,14 +3715,42 @@ cdef class RandomState:
         >>> x.shape
         (3, 3, 2)
 
+        Setting mode='fast' uses Cholesky decomposition which is fast but
+        not as robust as the default method. The mode='legacy' is the old
+        method using singular value decomposition to factorize the covariance
+        matrix, and is much slower than all other modes.
+
+        >>> y = np.random.multivariate_normal(mean, cov, (3, 3), mode='fast')
+        >>> y.shape
+        (3, 3, 2)
+
+        One can also supply a pre-computed factor of the covariance matrix.
+        This helps skip the step of factorizing the covariance every time the
+        function is called, resulting in speed-ups during repeated calls using
+        the same covariance matrix.
+
+        >>> cov = [[2, 2], [2, 2]]
+        >>> factor = np.linalg.cholesky(cov)
+        >>> np.random.multivariate_normal(mean, factor, (3, 3), mode='factor')
```

Comment:
This line is run via `python runtests.py --refguide-check` and is failing with a `ValueError`